server/files/stylesheets/docs.css

```diff
@@ -1616,7 +1624,6 @@ code:hover .django .tag .keyword {
   color: #892A6F;
   font-weight: normal;
 }
-
```

Comment:
Please keep the empty line before a comment block

Reply:
corrected the mistakes you can look again please